src/osx/webrequest_urlsession.mm

```diff
@@ -506,7 +506,7 @@ long WXUNUSED(dataTimeoutMs))
 {
     [m_session release];
     [m_delegate release];
-#if !wxOSX_USE_IPHONE
+#if !defined(wxOSX_USE_IPHONE) || !wxOSX_USE_IPHONE
```

Comment:
Could we simplify these checks to perform just one test, e.g. `#ifndef __WXDARWIN_OSX__`? Sorry to insist on it, but this would really make things simpler for people not very familiar with this code (such as me) to modify it in the future.